requirements/doc.txt

```diff
@@ -30,7 +30,7 @@ django==3.2.14
     # via
     #   -c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
     #   -r requirements/test.txt
-doc8==0.11.2
+doc8==1.0.0
```

Comment:
Only major change is drop of support for Python 3.6
https://github.com/PyCQA/doc8/releases/tag/v1.0.0